my-app/src/App.css

```diff
@@ -45,3 +55,47 @@ div#new-board-form {
   border: 0;
   background: 0;
 }
+
+#new-board-btn {
+  cursor: pointer;
+  font-family: "Montserrat", "Helvetica Neue", Arial, sans-serif;
+  font-weight: bold;
+}
+
+#personal-boards .board-btn {
+  color: white;
+  text-transform: none;
+  padding-top: 34px;
+}
+
+#personal-boards .dashboard-board-tools {
+  position: relative;
+  width: 100%;
+  height: 40px;
+  border-radius: .3rem;
+  margin-bottom: -40px;
+  background: #f5f5f5;
+  border: 1px solid #dadada;
+  border-top: none;
+}
+
+#personal-boards .dashboard-board-tools .tool-btn {
+  position: relative;
+  height: 100%;
+  width: 21%;
+  font-size: 1.2rem;
+  border: 0;
+  background: 0;
+  margin: 0 2%;
+  color: #333;
+}
```

Comment:
Here too - Alphabetical order

Reply:
Do you mean put the class names in alphabetical order, or each attribute? Like background, then color, then margin?